providers/PaperProvider/build.gradle

```diff
@@ -0,0 +1,4 @@
+dependencies {
+    implementation project(':providers:BaseProviders')
+    compileOnly 'com.destroystokyo.paper:paper-api:1.16.3-R0.1-SNAPSHOT'
```

Comment:
1.16.4?

Reply:
Doesn't need to be updated until we need a new method for paper provider